src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -377,6 +410,8 @@ const DirectoryTreeContextualMenu: React.FC<DirectoryTreeContextualMenuProps> =
                 );
             case DialogsId.ADD_NEW_CASE:
                 return <CreateCaseDialog open={true} onClose={handleCloseDialog} />;
+            case DialogsId.MOVE:
+                return <MoveDialog open={true} onClose={handleMoveDirectory} itemsCount={1} />;
```

Comment:
Ok to use this component, but you will need to customize the `title` I think, and you need to add an option to move a directory at the root level, as describe in the ticket.

Reply:
`onClose` should accept something like this as well I think `(node: TreeViewFinderNodeProps) => void;`, maybe adapt type definition in commons-ui, something like that : 
```
export interface TreeViewFinderProps {
  ...
  onClose: (nodes: TreeViewFinderNodeProps[] | TreeViewFinderNodeProps) => void;
  ...
```

Reply:
need to grey out the name of the folder being moved  preventing its selection while allowing the user to choose a new folder